packages/core/src/components/EditableHeading/__stories__/EditableHeading.stories.module.scss

```diff
@@ -12,5 +12,9 @@
     .editableHeading {
       width: 200px;
     }
+
+    .editableHeading:first-child {
+      width: 125px;
```

Comment:
Why this change?

Reply:
@shaharzil

Reply:
Because it gets really ugly with the addition of a fourth category